appveyor.yml

```diff
@@ -0,0 +1,25 @@
+version: 0.10.{build}

```

Comment:
what is this `version`?

Reply:
It is just some dummy version that AppVeyor needs to identify builds. We may want to sync it with the `dev` versions we have, or may not.